src/js/server/apply/router.js

```diff
@@ -87,13 +88,39 @@ applyRouter.post('/team', fileUploadMiddleware.none(), (req, res, next) => {
   });
 });
 
+// Process the RSVP response
+applyRouter.post('/rsvp', auth.requireAuth, function(req, res) {
+  const rsvp = req.body.rsvp;
+  if (rsvp) {
+    req.user.getHackerApplication().then(hackerApplication => {
+      if (hackerApplication == null) {
+        return Promise.resolve(null);
+      } else {
+        return hackerApplication.getApplicationResponse();
+      }
+    }).then(applicationResponse => {
+      if (applicationResponse != null) {
+        return rsvpToResponse(applicationResponse, rsvp);
```

Comment:
I think checking for existing RSVPs probably belongs with the business logic. Could explicitly add it there rather than just leaning on the unique key constraint